README.md

```diff
@@ -148,13 +22,14 @@ BSP](https://www.yoctoproject.org/downloads/bsps/pyro23/intel-corei7-64)
 * Minnowboard Max, Turbot
 
 ### PELUX Raspberry Pi 3
-Reference instance for Raspberry Pi 3 (coming soon)
+Reference instance for Raspberry Pi 3. Currently targets 32-bit mode.
```

Comment:
This is a change beyond what the commit message says. Although I agree this is nit picking it is still bad habit!

Reply:
I'll separate that into another commit.

Reply:
Updated now.